docs/devguide/metricset-details.asciidoc

```diff
@@ -112,7 +114,10 @@ Here an example for the `fields.yml` file from the MySQL module.
 include::../../metricbeat/module/mysql/_meta/fields.yml[]
 ----
 
-The last `fields.yml` file under `module/{module}/{metricset}/_meta/fields.yml` that contains all fields retrieved
+[float]
+===== metricbeat/{module}/{metricseat}/_meta/fields.yml  
+
+The last `fields.yml` file under `metricbeat/module/{module}/{metricset}/_meta/fields.yml` that contains all fields retrieved
```

Comment:
The phrasing of this sentence is still wrong. Should say "The last...file is under...and it contains... (otherwise it's a sentence fragment and doesn't match style/structure of other sections).

Reply:
Sorry again :)